### DISCUSSION TITLE
Using Talkie with Tone library

### DISCUSSION THREAD

**Discussion by ewlie** [ArminJo/Talkie#35] [General]
Hi Armin,

I'm trying to use Talkie (1.3.1) and Tone (by bhagman) together.

Your documentation indicates that this should be possible as long as you don't try to talk and tone at the same time:
"Added compatibility to Arduino Tone library by stopping timer1 interrupts at every end of speech"

No matter what I try, I run into conflicts on build like:
Tone.cpp.o (symbol from plugin): In function `timer0_pin_port':
(.text+0x0): multiple definition of `timer0_pin_port'

I've looked for alternative tone libraries to no avail and looked for compiler switches, but no luck so far.

Is it possible to output speech and tones within the same program (not at the same time)?

Do you have an example as a demonstration?

Am I using the correct Tone library?

Thanks for the excellent work BTW.

I'm using an Arduino UNO.

Thanks
John

**Comment by ArminJo** [comment]
Seems that you have 2 times the Tone.cpp file in your project. Remove talkie and you will likley still have this problem.
An example, which indirectly uses Tone can be found here https://github.com/ArminJo/PlayRtttl/blob/master/examples/PlayChristmasMelodyUSDistance/PlayChristmasMelodyUSDistance.ino.
